reflex-common/reflex-common/src/main/python/mcenter_cli/parallelm/mcenter_cli/execution_environment.py

```diff
@@ -19,6 +19,17 @@ def del_if_exists(d, field):
         del(d[field])
 
 
+def ee_get_json(mclient, ee_name):
+    all_ees = mclient.ee_list()
+    ee = list(filter(lambda x: x["name"] == ee_name, all_ees))
+    if len(ee) > 1:
+        raise Exception("More then one EE with the same name!!!!")
```

Comment:
I think it should be "than"
Aslo, as we are facing customers (or open source), let's discuss warning messages guidelines. I think one ! is enough ;)

Reply:
I agree with Yakov